modules/gateway/gateway.go

```diff
@@ -63,6 +66,8 @@ func (g *Gateway) Close() error {
 // Bootstrap handles mutexes manually to avoid having a lock during network
 // communication.
 func (g *Gateway) Bootstrap(bootstrapPeer modules.NetAddress) (err error) {
+	g.log.Println("INFO: initiated bootstrapping to", bootstrapPeer)
```

Comment:
INFO, WARN, ect. should all be constants. Probably established in the build package.